content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,46 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+<!-- keywords: data science project architecture, machine learning project architecture, machine learning workflow, data science workflow, machine learning file system, data science file system, data science project structure, machine learning project structure, notebook version control -->
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+Most of the time it needs to convert the the format of this data into a form
+that is required by the training models and supplying to data science / machine
+learning workflows. Sometimes it needs to be split into multiple files or
```

Comment:
OK to have some context but it should probably be a short intro paragraph only? Maybe just say something about project structures.

Reply:
BTW try to use other tooltips when possible e.g. `<abbr>project</abbr>` in this case.

Reply:
I think in BC documents, we need to make separation of concerns. If our aim is a quick introduction to DVC concepts, we don't need such *story material* to begin with. This was solely for SEO purposes. 

IMHO I can write blog posts / tutorials and tell stories with all kinds of SEO phrases there instead of trying to use these in docs. As you can see it's possible but I don't think it increases the quality of docs. 

I can paraphrase into a single paragraph or remove them completely. I'm fine with all.

Reply:
I didn't want to put `<abbr>project</abbr>` links to these as they _may_ (or may not) divert the reader. I was intending to define workspace first and then tell them about *DVC project.* 

![Screen Shot 2021-02-16 at 19 46 30](https://user-images.githubusercontent.com/476310/108094773-3b742100-7090-11eb-966e-bf67a81a02e9.png)

I'm putting <abbr> link to the last of these highlighted _project_ words.

Reply:
> we need to make separation of concerns. If our aim is a quick introduction to DVC concepts, we don't need such story material

We do already have tooltips for shot definitions. But you can't search for "pipeline dvc" on Google and find a tooltip (i.e. we want to have landing pages for our docs).

Another goal of these is to extract explanations out of the cmd refs (e.g. in that search example you find https://dvc.org/doc/command-reference/dag which shouldn't be where pipelines are explained)

In my mind most concept pages are 2-3 paragraph long in general, I think (we'll see as they come). There could also be some cool diagrams potentially (feel free to contribute sketches, we can fix them up later). They can still have a story, maybe just a very straightforward one? (motivation -> explanation -> uses/implications).
WDYT @shcheklein ?

> I'm putting link to the last of these highlighted project words.

I usually "tooltipy" just one instance of the term per page or big section, but I try to make it the first or 2nd one (as long as it doesn't distract e.g. no other links are nearby).

Reply:
p.p.s `straightforward one? (motivation -> explanation -> uses/implications` - I see that's more or less the structure here already 👍